.ko.yaml

```diff
@@ -2,3 +2,11 @@ baseImageOverrides:
   github.com/tektoncd/pipeline/cmd/creds-init: gcr.io/tekton-nightly/github.com/tektoncd/pipeline/build-base:latest
   github.com/tektoncd/pipeline/cmd/git-init: gcr.io/tekton-nightly/github.com/tektoncd/pipeline/build-base:latest
   github.com/tektoncd/pipeline/cmd/entrypoint: busybox  # image must have `cp` in $PATH
+baseBuildOverrides:
+  github.com/tektoncd/pipeline/cmd/controller:
+    env:
+      - name: CGO_ENABLED
+        value: 1
+    flags:
+      - name: ldflags
+        value: "-X github.com/tektoncd/pipeline/pkg/version.PipelineVersion=$(git describe --abbrev=0 --tags)"
```

Comment:
This means that at anytime during the development of version N, binaries built and deployed with ko will be marked with version N-1, which is not correct, since it's version N-1 plus all the ongoing development.